frame/support/procedural/src/pallet/parse/mod.rs

```diff
@@ -62,6 +66,25 @@ pub struct Def {
 }
 
 impl Def {
+	fn mark_benchmarking_cached_calls(storage_def: &mut StorageDef, item: &mut syn::Item) {
+		if let syn::Item::Type(typ) = item {
+			for attr in typ.attrs.as_slice() {
+				if attr.style == AttrStyle::Outer {
+					if let Some(seg) = attr.path.segments.last() {
+						if let Ok(_) = syn::parse2::<keyword::benchmarking>(seg.to_token_stream()) {
```

Comment:
If we make this function return a `syn::Result`, we can reduce the indentation here:
```suggestion
						seg.parse::<keyword::benchmarking>()?;
```